### DISCUSSION TITLE
Zoom Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by boredwisdom** [phase1geo/Minder#278] [Ideas]
zoom keyboard shortcuts does not work with numpad keys
also can <kbd>ctrl</kbd> + <kbd>/</kbd> be mapped for fit to screen

**Comment by phase1geo** [comment]
I believe that all zoom functions are already tied to keyboard shortcuts.

Control + (or =] to zoom in
Control - to zoom out
Control 0 (zero) to zoom to actual size
Control 1 (one) to do a zoom to fit

**Reply by boredwisdom** [reply]
zoom to fit : <kbd>ctrl</kbd>  + <kbd>1</kbd> is not working in my Ubuntu 20.04 machine
also can opening & closing of Properties drawer be mapped some keyboard shortcut ?

**Reply by phase1geo** [reply]
What version of Minder are you running? The properties sidebar can be hidden shown with a keyboard shortcut as well. If you click on the gear icon in the header bar and choose the keyboard shortcuts viewer, you'll see a listing of all of the keyboard shortcuts available.
